packages/odyssey-storybook/src/components/odyssey-mui/Checkbox/Checkbox.stories.tsx

```diff
@@ -261,6 +262,17 @@ export const Hint: StoryObj<typeof Checkbox> = {
   },
   play: async ({ canvasElement, step }) => {
     await checkTheBox({ canvasElement, step })("Checkbox Hint");
+    // await step("has visible hint", async () => {
```

Comment:
currently fails because the hint is included in the label, which it should not 
<img width="1489" alt="Screenshot 2024-07-25 at 3 55 42 PM" src="https://github.com/user-attachments/assets/ecbe280e-87e1-42fd-8bf2-33cd7b9c9ff0">

Reply:
@bryancunningham-okta We'll want this noted in a ticket.

For some reason, the hint is setup as part of the label and not the description in Checkbox.

Reply:
@AnirudhMani-okta Can you put a comment above it saying "This is commented because there's a separate ticket to cover this work: OKTA-XXXXXX".

Reply:
Addressed, added ticket: https://oktainc.atlassian.net/browse/OKTA-793465